keyboards/mexsistor/ludmila/rules.mk

```
@@ -0,0 +1,25 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader selection
+BOOTLOADER = caterina
```

You typically only see Caterina bootloader on Pro Micros, but since your controller is on-board, this is probably:

```suggestion
BOOTLOADER = atmel-dfu
```

---

Hi @noroadsleft, I didn't know that. I thought that when flashing the PCB as Pro Micro, in the "Bootloader" part I had to write Caterina. Even when generating Rules.mk it had atmel-dfu by default and I changed it to Caterina.

---

Did you ISP flash the chips when you got the PCBs in? Nearly every ATmega32U4 chip has the Atmel DFU bootloader installed at the factory, and the only way to change the bootloader is to ISP flash.

---

Yes, it did.

---

Which .hex file did you flash when you ISP'd the boards?

---

@noroadsleft  After run the "make mexsistor/ludmila:default" on  MSYS2 terminal, I open the QMK Toolbox and pick the "mexsistor_ludmila_default.hex" file stored in ".build" folder.

---

Okay that file wouldn't change the bootloader, so this is likely `atmel-dfu` bootloader. If we find the `RESET` keycode doesn't work at some point, we can change this.

---

Thanks @noroadsleft